### DISCUSSION TITLE
How to delete a single row of data on the dbeaver interface

### DISCUSSION THREAD

**Discussion by ffbh123456** [dbeaver/dbeaver#10389] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version
macosx
- DBeaver version
7.2.0.202008302047

#### Connection specification:
- MySQL 5.7.30

#### Describe your question:
I want to delete a row of data in the interface, but I can’t find a delete button. Does dbeaver support deleting data in the interface? I don't want to delete data through SQL Editor
<img width="499" alt="截屏2020-11-17 上午9 55 16" src="https://user-images.githubusercontent.com/20750357/99332145-174f4b00-28bb-11eb-9a3f-471d451f4130.png">

**Comment by kseniiaguzeeva** [comment, marked as answer]
Yes, please find the **delete current row** button on the bottom toolbar
https://github.com/dbeaver/dbeaver/wiki/Data-Editor
